index.js

```diff
@@ -1,6 +1,6 @@
 console.log("Hello Autoencoder 🚂");
 
-import * as tf from "@tensorflow/tfjs-node";
+import * as tf from "@tensorflow/tfjs-node-gpu";
```

Comment:
Do I need to install CUDA or anything like that to run tf.js with node GPU?

Reply:
yes, TensorFlow GPU only supports cuda. 

From my Python experience it needs an extremely specific version of cuda, but for JS it worked with out of the box 11.2 version. I only used the GPU version to speed up training, the CPU version works fine.